source/core/slang-io.h

```diff
@@ -50,6 +50,11 @@ namespace Slang
         static SlangResult GetPathType(const String & path, SlangPathType* pathTypeOut);
 
         static SlangResult GetCanonical(const String & path, String& canonicalPathOut);
+
+            /// Returns the executable path
+            /// @return The path in platform native format. Returns empty string if failed.
+        static String GetExecutablePath();
```

Comment:
I assume the naming here is to be consistent with the other functions? I wonder if we should go ahead and make all new functions follow the `lowerCamelCase` guidelines with the intention of migrating the other functions as we get a chance.

Either way is fine by me, since I understand the desire for local consistency.

Reply:
I think we should bite the bullet and change to the convention - lowerCamel. Just doing one function that way seems a bit odd, but say doing a class all in one go (like all of Path::) doesn't seem unreasonable.

Reply:
Yeah, I think the level of churn in the codebase is low enough right now (no major refactorings going on or anticipated soon) that doing class-at-a-time renaming PRs could work.

At that point we just need to decide when it is a priority to do that work. Maybe a reasonable start is that before diving into adding functionality to a class like `Path`, we first do a cleanup PR that brings it into line with the naming conventions?

Reply:
Ok sounds good.